Airports/EGCM/Basic.txt

```diff
@@ -1,3 +1,3 @@
 Leeds East
 N053.50.03.950 W001.11.43.470
-122.925
+126.500
```

Comment:
As of AIRAC 2303 the EGCM Radio frequency changes to 120.710, so closest would be 120.700, which is used at Lydd Approach, 125.725 is used at Valley ATIS. I don't know which frequency we should use for EGCM?